terraform/monitoring/main.tf

```diff
@@ -352,7 +368,7 @@ resource "grafana_dashboard" "at_a_glance" {
               "uid" : grafana_data_source.prometheus.uid
             },
             "exemplar" : true,
-            "expr" : "sum(rate(registered_clients{}[$__rate_interval]))",
+            "expr" : "sum(rate(registered_clients_total{}[$__rate_interval]))",
```

Comment:
After #334 this name has changed

<img width="773" alt="image" src="https://github.com/WalletConnect/push-server/assets/1979423/07c60027-09ff-4391-b9c7-0c0bffaa94ac">